sdks/java/src/main/java/com/google/agones/java/AgonesJavaSDK.java

```diff
@@ -0,0 +1,88 @@
+package com.google.agones.java;
+
+import com.google.agones.java.SDKGrpc.SDKBlockingStub;
+import com.google.agones.java.SDKGrpc.SDKStub;
+import com.google.agones.java.observer.EmptyObserver;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.stub.StreamObserver;
+
+public class AgonesJavaSDK {
+
+    private static final String DEFAULT_HOST = "127.0.0.1";
```

Comment:
As I see in other SDKs we are using localhost here. Example from Node SDK:
```
		this.client = new services.SDKClient('localhost:59357', grpc.credentials.createInsecure());
```

Reply:
Changed.